src/components/ToolMenu/LayerTree/index.tsx

```diff
@@ -61,6 +62,7 @@ export type LayerTileLoadCounter = {
 export type LayerTreeConfig = {
   enabled?: boolean;
   activeUploadTools?: UploadTools[];
+  showLegends?: boolean;
 };
```

Comment:
Not directly related to the changes in this PR, but it looks like the type `LayerTreeConfig` is used in the store only. Probably we should think about moving it over.

Reply:
Moved the type LayerTreeConfig to the store